### PR TITLE
[SFT-552] - Add convert feature to Predefined options/Element options to Custom options

### PR DESCRIPTION
**Pull request by seandelaney**

### CODE REVIEW

**Review by kjmartens — APPROVED** [review]
From a visual and behavior standpoint, it looks good to me. Good work! 🙂

File: packages/client/src/app/components/form-controls/control-types/options/options.tsx [hunk 1]
```diff
@@ -42,9 +45,16 @@ const Options: React.FC<ControlType<OptionsProperty, Field>> = ({
     );
   };
 
+  const convertToCustomValues = (): void =>
+    updateValue({
+      source: Source.Custom,
+      useCustomValues: true,
+      options: options.map((option: Option) => option),
```
Comment (gustavs-gutmanis) [inline packages/client/src/app/components/form-controls/control-types/options/options.tsx]: This still returns the actual references unfortunately,
Better do this

```suggestion
      options: [...options],
```

**Review by gustavs-gutmanis — APPROVED** [review]
Very solid and nice work!

File: packages/client/src/app/components/form-controls/control-types/options/options.tsx (hunk 1 above)
Comment (seandelaney) [reply packages/client/src/app/components/form-controls/control-types/options/options.tsx:52]: Just wanted to test you... ;)

Comment (seandelaney) [reply packages/client/src/app/components/form-controls/control-types/options/options.tsx]: @gustavs-gutmanis Are you sure? Typescript not happy

 ```
TS2322: Type '(Option | OptionGroup)[]' is not assignable to type 'Option[]'.
  Type 'Option | OptionGroup' is not assignable to type 'Option'.
    Property 'value' is missing in type 'OptionGroup' but required in type 'Option'.

```

I can do this

`options: [...options] as Option[],`

**Review by gustavs-gutmanis — APPROVED** [review]

**Pull request merged**